packages/suite/src/views/dashboard/AssetsView/AssetTable/AssetRow.tsx

```diff
@@ -10,25 +10,56 @@ import {
     Translation,
     TrendTicker,
 } from 'src/components/suite';
-import { isTestnet } from '@suite-common/wallet-utils';
+import { getAccountTotalStakingBalance, isTestnet } from '@suite-common/wallet-utils';
 import { goto } from 'src/actions/suite/routerActions';
-import { useAccountSearch, useDispatch } from 'src/hooks/suite';
+import { useAccountSearch, useDispatch, useSelector } from 'src/hooks/suite';
 import { spacings } from '@trezor/theme';
-import { AssetFiatBalance } from '@suite-common/assets';
 import { AssetCoinLogo } from '../AssetCoinLogo';
 import { AssetCoinName } from '../AssetCoinName';
 import { CoinmarketBuyButton } from '../CoinmarketBuyButton';
+import { BigNumber } from '@trezor/utils';
+import { TokenInfo } from '@trezor/blockchain-link-types';
+import { AssetTokenRow } from './AssetTokenRow';
+import { selectCoinDefinitions } from '@suite-common/token-definitions';
+import {
+    enhanceTokensWithRates,
+    getTokens,
+    sortTokensWithRates,
+} from 'src/utils/wallet/tokenUtils';
+import { selectAssetAccountsThatStaked } from '@suite-common/wallet-core';
+import { Account, RatesByKey } from '@suite-common/wallet-types';
+import { AssetStakingRow } from './AssetStakingRow';
+import { AssetFiatBalance } from '@suite-common/assets';
+import { FiatCurrencyCode } from '@suite-common/suite-config';
+import { AssetTableExtraRowsSection as Section } from './AssetTableExtraRowsSection';
+import { TokenIconSetWrapper } from 'src/components/wallet/TokenIconSetWrapper';
 
-interface AssetTableProps {
+export interface AssetTableRowProps {
     network: Network;
     failed: boolean;
-    cryptoValue: string;
-    isLastRow?: boolean;
+    assetNativeCryptoBalance: string;
+    stakingAccounts: Account[];
+    assetTokens: TokenInfo[];
+    isStakeNetwork?: boolean;
     assetsFiatBalances: AssetFiatBalance[];
+    accounts: Account[];
 }
 
 export const AssetRow = memo(
-    ({ network, failed, cryptoValue, assetsFiatBalances }: AssetTableProps) => {
+    ({
+        network,
+        failed,
+        assetNativeCryptoBalance,
+        assetTokens,
+        stakingAccounts,
+        assetsFiatBalances,
+        localCurrency,
+        currentFiatRates,
+        accounts,
+    }: AssetTableRowProps & {
+        localCurrency: FiatCurrencyCode;
+        currentFiatRates?: RatesByKey;
+    }) => {
```

Comment:
why?

Reply:
Understandable, I got rid of this
https://github.com/trezor/trezor-suite/pull/14996/commits/db24b70b62881dce04ca81e408a9e95506a2cc3a